include/sapi/tss2_tcti.h

```diff
@@ -165,6 +165,23 @@ TSS2_TCTI_POLL_HANDLE *handles, size_t *num_handles);
 
 typedef TSS2_TCTI_CONTEXT_COMMON_V1 TSS2_TCTI_CONTEXT_COMMON_CURRENT;
 
+#define TCTI_INFO_SYMBOL "Tss2_Tcti_Info"
+
+typedef TSS2_RC (*TSS2_TCTI_INIT_FUNC) (
+    TSS2_TCTI_CONTEXT *tctiContext,
+    size_t *size,
+    const char *config
+    );
+
+typedef struct {
+    const char *name;
+    const char *description;
+    const char *config_help;
+    TSS2_TCTI_INIT_FUNC init;
+} TSS2_TCTI_INFO;
+
+typedef TSS2_TCTI_INFO* (*TSS2_TCTI_INFO_FUNC) (void);
```

Comment:
I was wondering if this can/may/should be
```
typedef const TSS2_TCTI_INFO* (*TSS2_TCTI_INFO_FUNC) (void);
```

Would also have to be changed in all instatiations below...

Reply:
works for me. PR updated.